### DISCUSSION TITLE
Is it possible to disable the 9.5 persistent caching for page bundels?

### DISCUSSION THREAD

**Discussion by moop-moop** [vercel/next.js#17286] [Help] [answered]
I know this may be a fringe case, but is it possible to disable the persistent caching for page bundles that was added to Next.js 9.5?
https://nextjs.org/blog/next-9-5#persistent-caching-for-page-bundles

I am actually trying to configure a next build with as few unique hashes and paths as possible.

**Comment by lfades** [comment, marked as answer]
No. It's not possible, we do that in order to serve static and cache static files in the browser and to invalidate them when needed.
